test/unit/modules/keyboard.coffee

```diff
@@ -80,5 +80,20 @@ describe('Keyboard', ->
       expect(dom($('.ql-bold').get(0)).hasClass('ql-active')).toBe(true)
       expect(dom($('.ql-size').get(0)).value()).toBe(size)
     )
+
+    it('removeHotkey by name', ->
+      fn = @quill.getModule('keyboard').removeHotkey('BOLD')
+      expect(typeof fn).toBe('function');
+    )
+
+    it('removeHotkey by number', ->
```

Comment:
Does that mean we would require you to specify `removeHotkey({ key: 13 })` instead of `removeHotkey(13)`? This doesn't seem to parallel well with the shortcut of providing a string.